jelly/src/main/java/org/kohsuke/stapler/jelly/HTMLWriterOutput.java

```diff
@@ -64,18 +67,15 @@ private HTMLWriterOutput(HTMLWriter hw, OutputFormat fmt, boolean escapeText) {
         hw.setEscapeText(escapeText);
         this.htmlWriter = hw;
         this.format = fmt;
+        format.setExpandEmptyElements(true);
     }
 
     @Override public void close() throws IOException {
         htmlWriter.close();
     }
 
-    /**
-     * False to turn off HTML handling and reenable {@code />} for any empty XML element.
-     * True to switch back to default mode with HTML handling.
-     */
+    @Deprecated
     public void useHTML(boolean enabled) {
-        htmlWriter.setEnabled(enabled);
-        format.setExpandEmptyElements(enabled);
+        LOGGER.fine("Deprecated: HTMLWriterOutput.useHtml() has been deprecated and its functionality removed.");
```

Comment:
```suggestion
        LOGGER.log(Level. WARNING, null, new UnsupportedOperationException("Deprecated: HTMLWriterOutput.useHtml() has been deprecated and its functionality removed."));
```

since no one is going to see this message otherwise.